### ISSUE TITLE
Bulk hide environments

### ISSUE THREAD

**Issue by Numynum** [getsentry/sentry#59270]
### Problem Statement

We have some projects, where we have close to 100k environments.
In the past, we wanted to mark all environments as hidden, to get a clean slate to start from; but currently, this is only possible, by hiding the environments 1 by 1. This is a rather lengthy process, even when automating it by using the (undocumented) API endpoint for this.

We would like there to be a button and/or endpoint, to mark all environments as hidden.
This is also likely going to improve our performance, even though #58790 has been a huge improvement already.

### Solution Brainstorm

Basically a bulk version of the endpoint defined here https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_environment_details.py#L35 would be lovely

On the UI side of things, you could go a route similar to the issues list, where there's a checkbox, to select multiple environments, and another option, to select all, and a button to hide selected.

The current UI requires a full refresh of the environments list, for each environment you hide, which is rather slow

### Product Area

Projects - Project Details

**Comment by sentaur-athena** [comment]
Thank you for the feedback. Adding to the backlog so the team can prioritize in the future.

**Comment by dmitrydvornichenko** [comment]
Hello sentry team.
I have the same request.
We have dynamic development environments, which live untill a feature is merged into master. But in sentry they stay forever.
So, there should be the way to hide an environment by script.
Could you please create an api endpoint to hide an environment?
That will be a very useful feature! (everyone says this when requesting a feature, I think ;-) )

**Comment by carpe-diem** [comment]
Here with the same problem, should be great to have a way to hide environments programmatically, either through the API or by implementing some code-based solution. Our goal is to automatically hide environments once a preview environment is taken down.

**Comment by mhornbacher** [comment]
+1 😄
